test/scripts/browser-test/run-test.mjs

```diff
@@ -0,0 +1,230 @@
+/**
+ * This script is served by `harness.mjs` and runs in the browser.
+ */
+import { WASI, File, OpenFile, ConsoleStdout, PreopenDirectory } from 'https://cdn.jsdelivr.net/npm/@bjorn3/browser_wasi_shim@0.3.0/+esm'
+import { polyfill } from 'https://cdn.jsdelivr.net/npm/wasm-imports-parser@1.0.4/polyfill.js/+esm';
```

Comment:
It seems like we are taking a dependency here this WASI implementation.    Maybe this should be documented somewhere?   What happens if this breaks?   Who maintains this WASI implementation?

Reply:
Added a sentence about the dependency to test/README.md: https://github.com/WebAssembly/wasi-libc/pull/576
If the dependency breaks, wasi-libc's CI can fail, but nothing affects the users of wasi-libc. Although it's maintained under [a personal namespace](https://github.com/bjorn3/browser_wasi_shim/), the implementation is the most reliable one on browser today as far as I know.

Reply:
FWIW, I see this browser testing infrastructure as useful but experimental. If we were to run into many problems consistently, I think it would make sense to rip it out.  (and I reviewed it with that in mind). But I'm hopeful that things are stable enough at this point that that won't be a problem.